elements/bulbs-pdfobject/bulbs-pdfobject.js

```diff
@@ -21,7 +21,7 @@ class BulbsPDFObject extends BulbsHTMLElement {
   embedPDF () {
     let poster = this.getAttribute('poster');
     let options = {
-      fallbackLink: `<a href='[url]'><div class="fallback-btn">Tap to explore document</div><img src="${poster}"></a>`,
+      fallbackLink: `<a href='[url]' data-track-category="SF: Trump Docs 2017" data-track-action="Embed Fallback" data-track-label='[url]'><div class="fallback-btn">Tap to explore document</div><img src="${poster}"></a>`,
```

Comment:
It's currently being used in liveblog which has it's own `data-track-category` value. So needs to be passed through the element to avoid bubbling up to the wrong category.  The issue is that this would need to be updated in 80+ places due to current Trump Docs being ingested. I'm fine with revisiting this  after launch.